packages/liquidator/src/liquidationStrategy.js

```diff
@@ -219,24 +204,24 @@ module.exports = (
   // Returns true if enough of the withdrawal liveness has passed that the WDF strategy can be activated.
   function passedDefenseActivationPercent({ position, currentBlockTime }) {
     return (
-      hasWithdrawRequestPending({ position, currentBlockTime }) &&
       withdrawProgressPercent(withdrawLiveness, position.withdrawalRequestPassTimestamp, currentBlockTime) >=
-        parseFloat(defenseActivationPercent)
+      parseFloat(defenseActivationPercent)
     );
   }
 
-  // Should we try to delay this withdrawal. Update this logic to change conditions to run delay strategy.
-  function shouldLiquidateMinimum({ position, syntheticTokenBalance, currentBlockTime }) {
```

Comment:
This function has too many branches and is the main reason why I think this refactor will make the class more readable.